pkg/genesis/genesis.go

```diff
@@ -23,10 +23,10 @@ func NewGenesis(
 	proposerAddress []byte,
 ) Genesis {
 	genesis := Genesis{
-		ChainID:              chainID,
-		GenesisDAStartHeight: genesisDAStartHeight,
-		InitialHeight:        initialHeight,
-		ProposerAddress:      proposerAddress,
+		ChainID:            chainID,
+		GenesisDAStartTime: genesisDAStartHeight,
```

Comment:
This likely needs to be updated since it expects a time not a height

Reply:
yes its the naming convention, in the adr its height but we never implemented it